lib/transforms/subsetFonts.js

```diff
@@ -539,7 +600,7 @@ module.exports = ({
                     return htmlAsset.addRelation({
                         type: 'HtmlPreloadLink',
                         hrefType: 'rootRelative',
-                        to: assetGraph.findAssets({ url: fontUsage.props.src })[0],
+                        to: assetGraph.findAssets({ url: fontUsage.fontUrl })[0],
```

Comment:
Actually, `to: fontUsage.url` should work :)

Reply:
Done